src/Security/Authenticator/GitHubAuthenticator.php

```diff
@@ -71,27 +72,38 @@ public function authenticate(Request $request): PassportInterface
                     return $existingUser;
                 }
 
-                /** @var User|null $user */
-                $user = $this->em->getRepository(User::class)
-                    ->findOneBy(['email' => $githubUser->getEmail()]);
+                $response = $this->httpClient->request('GET', 'https://api.github.com/user/emails', [
+                    'headers' => [
+                        'Authorization' => sprintf('token %s', $accessToken->getToken()),
+                    ],
+                ]);
```

Comment:
The email is no more in the received payload from GitHub? In the `GithubResourceOwner` object?

Reply:
If I understood the API correctly, the `GithubResourceOwner` only contains public emails, but not everyone has its email as public. With my account at least, the `email` was set to `null`